test/utils.js

```diff
@@ -138,15 +138,23 @@ const utils = {
     return receipt.logs[0].args.pollID;
   },
 
-  decimalDivide: (numerator, denominator) => {
+  divideAndGetWei: (numerator, denominator) => {
     const weiNumerator = Eth.toWei(BN(numerator), 'ether');
```

Comment:
Good changes, but this is still bothering me. It just doesn't feel right. Lets talk about it tomorrow.